onnxruntime/core/providers/qnn/builder/op_builder_factory.cc

```diff
@@ -51,6 +52,9 @@ OpBuilderRegistrations::OpBuilderRegistrations() {
     CreateSimpleOpBuilder("LogSoftmax", *this);
     CreateSimpleOpBuilder("MatMul", *this);
     CreateSimpleOpBuilder("Concat", *this);
+
+    CreateSimpleOpBuilder("QuantizeLinear", *this);
+    CreateSimpleOpBuilder("DequantizeLinear", *this);
```

Comment:
This is to handle a special case that some single DQ nodes not grouped with any other non-QDQ nodes. It's caused by some issues in LayoutTransformer + optimizer which still need to look into.
Existing QdqOpBuilder assumes single DQ nodes on graph output.